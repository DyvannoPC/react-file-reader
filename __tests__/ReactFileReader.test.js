import React from 'react';
import ReactFileReader from '../ReactFileReader';
import renderer from 'react-test-renderer';

test('the component renders', () => {
  const component = renderer.create(
    <ReactFileReader handleFiles={() => ''}>
      <p>Upload</p>
    </ReactFileReader>
  );

  let tree = component.toJSON();
  expect(tree).toMatchSnapshot();
});
