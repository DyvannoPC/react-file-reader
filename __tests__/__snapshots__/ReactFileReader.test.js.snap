// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`the component renders 1`] = `
<div
  className="react-file-reader"
>
  <input
    accept="image/*"
    className="react-file-reader-input"
    id="test-render"
    multiple={undefined}
    onChange={[Function]}
    style={
      Object {
        "left": "-99999999px",
        "opacity": "0px",
        "position": "fixed",
        "width": "0px",
      }
    }
    type="file"
  />
  <div
    className="react-file-reader-button"
    onClick={[Function]}
  >
    <p>
      Upload
    </p>
  </div>
</div>
`;
